tests/test_pipelines_fill_mask.py

```diff
@@ -111,7 +136,7 @@ def test_torch_fill_mask_results(self):
             "My name is <mask>",
             "The largest city in France is <mask>",
         ]
-        valid_targets = [" Patrick", " Clara"]
+        valid_targets = ["ĠPatrick", "ĠClara"]
```

Comment:
Those tests could stay the same, but it's actually missing from the vocabulary, so it's hitting the slow path and triggering warnings.

the 2 small models don't share the same vocabulary, so if we want tests that avoid the slow path all  the time we should have 2 version of the targets IMO.